src/pages/home/Home.jsx

```diff
@@ -1,9 +1,104 @@
-import React from 'react'
-
+import {React, useEffect, useState} from 'react';
+import axios from 'axios';
+import BackgroundIamge from "../../assets/background-image.png";
+import "../../../src/App.css";
+import {Link} from "react-router-dom";
 function Home() {
+    const [category, setCategory] = useState([]);
+
+    useEffect(() => {
+        (async () => {
+            try {
+                const response = await axios.get('/api/categories');
+                setCategory(response.data.categories);
+              } catch (error) {
+                console.error(error);
+              }
+        })();
+    },[]);
   return (
-    <div>Home</div>
-  )
+    <div>
+    <section className="home" id="home">
+        <div className="row">
+            <div className="content_for-bookhub">
+                <h3 className="style-head"><span>Reading made</span>
+                    <span className="style_text-yellow"><span className="style-yellow">better</span></span></h3>
+                <p> <span className="style-description">
+                        Read more books
+                        <br/>
+                        Meet great people
+                        <br/>
+                        Engage in meaningful conversations
+                    </span></p>
+                <Link to="/product" className="btn-bookooze">Bookstore</Link>
+            </div>
+
+            <div className="books-slider">
+                <div>
+                    <img src={BackgroundIamge} className="style-image" alt=""/>
```

Comment:
do not keep alt empty and other than these everything looks good !

Reply:
done
thank you for the suggestion